### DISCUSSION TITLE
Inquiry About Killer Applications Utilizing HPX

### DISCUSSION THREAD

**Discussion by zuoyanzhang** [STEllAR-GROUP/hpx#6524] [General]
Hello, HPX team. I am curious to know if there are any applications or projects currently utilizing HPX that can be considered "killer applications". Specifically, I'm looking for examples where HPX has made a significant impact in terms of performance, scalability, or solving complex problems that were not feasible with other technologies.

Thank you for your time and assistance.

**Comment by Pansysk75** [comment]
Hi @zuoyanzhang,
Here are some links that may be of interest:

- https://hpx.stellar-group.org/applications/
- https://github.com/STEllAR-GROUP/hpx/discussions/5368

**Comment by zuoyanzhang** [comment]
Many Thanks to you, Panos.
